shared/actions/profile/index.js

```diff
@@ -76,23 +77,29 @@ function* _onUserClick(action: Constants.OnUserClick): SagaGenerator<any, any> {
   if (!username.includes('@')) {
     yield put(switchTo([profileTab]))
     yield put(navigateAppend([{props: {username}, selected: 'profile'}], [profileTab]))
+    return
+  }
+
+  let props = {}
+  const searchResult = yield select(Selectors.searchResultSelector, username)
```

Comment:
I'm a bit confused about why we do this. We're filling in information from the last search result, which means that the displayed profile info is slightly different depending on if we've just searched for the clicked-on assertion. (e.g., `fullUsername`, and also whether or not the assertion is upgraded to a keybase ID above).

Why doesn't the profile page itself query the backend for user info?